packages/custom-functions-metadata/package.json

```diff
@@ -29,7 +29,7 @@
     "assert": "^1.5.0",
     "commander": "^6.2.0",
     "nconf": "^0.10.0",
-    "office-addin-cli": "^1.1.1",
```

Comment:
Do we need this downgrade?

Reply:
We don't, I just wanted to make the PR as readable as possible, so I chose to make package.json consistent with what's on package-lock.json, not the other way around. I'll change this to latest and update the branch.